dns/dns.tf

```
@@ -1,120 +1,80 @@
 #-------------------------------------------------------------------------------
-# Create all necessary DNS records for the IPA servers.
+# Create all necessary DNS records for the IPA cluster.
 # -------------------------------------------------------------------------------
 resource "aws_route53_record" "server_A" {
```

Should we make this corresponding name change (and update references to it)?
```suggestion
resource "aws_route53_record" "cluster_A" {
```

---

Please see commit 3d05b3233c99761a61ea43cd98017ca3dea773e5.